src/logging.rs

```diff
@@ -28,6 +28,7 @@ pub struct Logger<T> {
 }
 
 impl<T: Send + io::Write> Logger<T> {
+    #[cfg_attr(feature = "clippy", allow(new_ret_no_self))]
```

Comment:
Let's add a comment here pointing to the issue tracking this broken lint: https://github.com/rust-lang-nursery/rust-clippy/issues/734